website/docs/getting-started/setup/instance-configuration/authentication/github-login.md

```diff
@@ -7,63 +7,50 @@ sidebar_position: 2
-    * Configure `APPSMITH_SIGNUP_ALLOWED_DOMAINS` with a second-level domain name **`(ex: abc.com)`**, to only allow users with an email address from that domain name to log in **`(ex: john@abc.com)`**
-      * **Note**: you can provide several domain names using a comma-separated list
-    * [Restart the Appsmith instance](../)
-
----
+## Configure Appsmith instance  
 
 :::caution
-The Client ID and Client Secret entered aren't verified by any means. Please test it out manually before disabling the Form login authentication method. If Form login is disabled before testing the GitHub authentication configuration setup, it may lead to the user getting stuck on the login screen if the GitHub configuration is invalid. You’ll have to update the environment variables manually to fix this, as you can't update it via the Admin settings page since you won’t be logged in anymore
+Test the Client ID and Client Secret before disabling Form login on Appsmith. This ensures that users can access Appsmith with their GitHub credentials. To fix an invalid GitHub configuration, update the environment variables.
 :::
-
-After these steps, GitHub Login should now be enabled for your Appsmith installation. If you are running appsmith behind an ELB / Proxy, please configure Nginx [following this guide](/help-and-support/troubleshooting-guide/deployment-errors#oauth-sign-up-not-working).
\ No newline at end of file
+There are two ways to configure the self-hosted Appsmith instance - 
+- Admin Settings
+- Environment Variables
+
+**Admin Settings:** Click **Admin Settings > Authentication > Enable**(GitHub). Add the `Client ID` and  `Client Secret`. Click the **Save & Restart** button at the bottom of the page. 
+
+![](/img/as_github_auth_config.png)
+
+**Environment Variables:** Navigate to the `docker.env` file located in `<PROJECT_ROOT>/stacks/configuration/`. Update the values for the following keys:
+
+* Configure the `APPSMITH_OAUTH2_GITHUB_CLIENT_ID` and `APPSMITH_OAUTH2_GITHUB_CLIENT_SECRET` fields with the client ID and client secret generated in the step above.
+* Configure `APPSMITH_SIGNUP_ALLOWED_DOMAINS` with a second-level domain name (ex: `abc.com`), to only allow users with an email address from that domain name to log in (ex: `john@abc.com`). You can provide several domain names using a comma-separated list.
+    
+  ```bash
+  # Example configuration in docker.env file
+  # ********* GitHub OAuth **********
+  APPSMITH_OAUTH2_GITHUB_CLIENT_ID=YOUR_GITHUB_CLIENT_ID
+  APPSMITH_OAUTH2_GITHUB_CLIENT_SECRET=YOUR_GITHUB_CLIENT_SECRET
+  APPSMITH_SIGNUP_ALLOWED_DOMAINS=exampledomain.com
+  # ******************************
+   ```
+* Restart the Appsmith instance. 
+
+After these steps, your Appsmith installation should now enable GitHub Login. If you are running appsmith behind an ELB / Proxy, please configure Nginx [following this guide](/help-and-support/troubleshooting-guide/deployment-errors#oauth-sign-up-not-working).
```

Comment:
The ELB/Proxy piece appears to be a Troubleshooting guide

Reply:
Yes, it is. We kept it like this as it was a specific error.